doc/nrf/gs_installing.rst

```diff
@@ -365,88 +360,10 @@ Use the following commands to install the requirements for each repository.
-   For example, :file:`c:\\gccarmemb\\9_2019-q4-major`, where :file:`9_2019-q4-major` is the *version-folder* name edited to contain no empty spaces.
-#. If you want to build and program applications from the command line, define the environment variables for the GNU Arm Embedded Toolchain.
-   Depending on your operating system:
-
-    .. tabs::
-
-       .. group-tab:: Windows
-
-          Open a command-line window and enter the commands below.
-
-          If you did not install the toolchain in the recommended folder, change the value of :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the folder you used and make sure to provide the name of the *version-folder*.
-
-            .. parsed-literal::
-               :class: highlight
-
-               set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-               set GNUARMEMB_TOOLCHAIN_PATH=\ c:\\gnuarmemb\\version-folder
-
-       .. group-tab:: Linux
-
-          Open a terminal window and enter the commands below.
-
-          If you did not install the toolchain in the recommended folder, change the value of :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the folder you used and make sure to provide the name of the *version-folder*.
-
-            .. parsed-literal::
-               :class: highlight
-
-               export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-               export GNUARMEMB_TOOLCHAIN_PATH=\ "~/gnuarmemb/*version-folder*"
-
-       .. group-tab:: macOS
-
-          Open a terminal window and enter the commands below.
-
-          If you did not install the toolchain in the recommended folder, change the value of :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the folder you used and make sure to provide the name of the *version-folder*.
-
-            .. parsed-literal::
-               :class: highlight
-
-               export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-               export GNUARMEMB_TOOLCHAIN_PATH=\ "~/gnuarmemb/*version-folder*"
-
-#. Set the environment variables persistently.
-   Depending on your operating system:
-
-    .. tabs::
-
-       .. group-tab:: Windows
-
-          Add the environment variables as system environment variables or define them in the :file:`%userprofile%\zephyrrc.cmd` file as described in :ref:`build_environment_cli`.
-          This lets you avoid setting them every time you open a command-line window.
-
-       .. group-tab:: Linux
-
-          Define the environment variables in the :file:`~/.zephyrrc` file as described in :ref:`build_environment_cli`.
-          This lets you avoid setting them every time you open a terminal window.
-
-       .. group-tab:: macOS
+Install a Toolchain
+*******************
 
-          Define the environment variables in the :file:`~/.zephyrrc` file as described in :ref:`build_environment_cli`.
-          This lets you avoid setting them every time you open a terminal window.
+To install a toolchain, refer to "Install a Toolchain" section in Zephyr's :ref:`zephyr:getting_started`.
```

Comment:
It would be nice to link to the Zephyr section directly. Could we add a label upstream later on and point directly to it?

Reply:
Yes, sure!

Reply:
As @gregersrygg suggested, I think it would be nice to include the instructions directly from Zephyr as we have been doing for other parts of getting started. Could you add the necessary changes @richabp?

Reply:
Ok. What are your thoughts on this @b-gent?

Reply:
Please use the ``.. ncs-include::`` RST element to include this content from Zephyr docs, like we do in the 'Install required tool section:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/_sources/gs_installing.rst.txt